docs/brain/brain.md

```diff
@@ -1,59 +1,133 @@
+        broker_ip: "127.0.0.1"
+        topic: "topic1"
+  neurons:
+    - say:
+        message:
+          - "The light is now {{ mqtt_subscriber_message }}"
+```
+
+### Available signals
+
+Here is a list of core signal that are installed natively with Kalliope
+
+| Name                                                   | Description                                                       |
+|--------------------------------------------------------|-------------------------------------------------------------------|
+| [event](signals/event)                     | Launch synapses periodically at fixed times, dates, or intervals. |
+| [mqtt_subscriber](signals/mqtt_subscriber) | Launch synapse from when receive a message from a MQTT broker     |
+| [order](signals/order)                     | Launch synapses from captured vocal order from the microphone     |
+| [geolocation](signals/geolocation)         | Define synapses to be triggered by clients handling geolocation   |
+
+See the full list of core and community signals on the [Kalliope's website](https://kalliope-project.github.io/signals_marketplace.html).
+
+## Neurons
 
 A neuron is a plugin that performs a specific action. You use it to create a synapse.
-You can add as many neurons as you want to a synapse. The neurons are executed one by one when the input order is triggered.
-If you want to install a community neuron, see the [neuron list documentation](neuron_list.md).
+You can add as many neurons as you want to a synapse. The neurons are executed one by one when one of the input signal is triggered.
+
+### Syntax
 
-## Usage
 Neurons are declared in the `neurons` section of a synapse in your brain file.
 The `neurons` section is a list (because it starts with a "-") which contains neuron modules names
-```yml
+```yaml
 neurons:
     - neuron_1_name
     - neuron_2_name
     - another_neuron
 ```
 
 Some neurons need parameters that can be passed as arguments following the syntax bellow:
-```yml
+```yaml
 neurons:
     - neuron_name:
         parameter1: "value1"
         parameter2: "value2"
 ```
+
+Eg:
+
 > **note:** parameters are indented with two spaces bellow the neuron's name following the YAML syntax.
 
 > **note:** Kalliope will try to load the neuron from your resources directory, then from core neuron packages.
 
-To know the list of required parameters, check of documentation of the neuron.
-Full list of [available neuron here](neuron_list.md)
+### Input values
 
-## Input values
+Neurons require some **parameters** from the synapse declaration to work. Those parameters, also called arguments, can be passed to the neuron from:
 
-Neurons require some **parameters** from the synapse declaration to work. Those parameters, also called arguments, can be passed to the neuron in two way:
-- from the neuron declaration
-- from global variables
-- from the captured order
+- the neuron declaration
```

Comment:
add kalliope_memory ?

Reply:
and define the priority order ? 
eg : if a have two parameters {{name}}, one coming from the signaln one defined in global variable ?

Reply:
kalliope memory added. 
And I don't know the priority. I'll need to check into the code.